pg/seed.pgsql

```diff
@@ -21,22 +21,29 @@ SET default_tablespace = '';
 SET default_table_access_method = heap;
 
 --
--- Name: tasks; Type: TABLE; Schema: public; Owner: postgres
+-- Name: animals; Type: TABLE; Schema: public; Owner: tpl1_2021h1
 --
 
-CREATE TABLE public.tasks (
+CREATE TABLE public.animals (
     id integer NOT NULL,
-    name text
+    name character varying,
```

Comment:
Googled postgres character varying datatype, I've always used VARCHAR which is just an alias for this! Nice! Would you consider adding a number to limit the characters people could input for these fields? Just in case someone was trying to put faulty data? Such as, name VARCHAR(255) or character varying(255)?